### PR TITLE
[Overflow-204] Display the Answers Component Dynamically

### PR DESCRIPTION
**Pull request by jescabillas**
## Backlog Link
https://framgiaph.backlog.com/view/SUN_OVERFLOW-204

## Commands
None

## Pre-conditions
None

## Expected Output
- Answer component will dynamically show the answer's data if the question has answer(s)
- Edit answer will only show to the creator of the answer

## Notes
None

## Screenshots
image

### CODE REVIEW

File: frontend/components/organisms/AnswerComponent/index.tsx [hunk 1]
```diff
@@ -31,19 +31,17 @@ const AnswerComponent = () => {
         reValidateMode: 'onSubmit',
     })
 
-    useEffect(() => {
-    }, [isDisableSubmit])
+    useEffect(() => {}, [isDisableSubmit])
```
Comment (kenorenzsun) [inline frontend/components/organisms/AnswerComponent/index.tsx]: Is this used ? please delete if it does nothing

**Review by jules-lucero — APPROVED** [review]

File: frontend/components/organisms/AnswerComponent/index.tsx (hunk 1 above)
Comment (jescabillas) [reply frontend/components/organisms/AnswerComponent/index.tsx]: This is not part of the changes for this PR

**Review by kenorenzsun — APPROVED** [review]

**Comment by presyos** [comment]
## Working
- [x] Display the answer

## Comment
- [ ] The answer is displayed but the format is not applied instead it was displayed as a plain text
- Steps to reproduce
 1. Add answers with the format
 2. Click Submit answer button
-------------------------------------------------------------------------------

image

-------------------------------------------------------------------------------
- Total Test Time: 0.5 hr

**Comment by presyos** [comment]
## Approved PR [#67](https://github.com/framgia/sph-sun-overflow/pull/67) 
- [x] Display the answers to the question

## Proof
----------------
[Screencast 2023-02-09 13-51-18.webm](https://user-images.githubusercontent.com/112840596/217729504-045b9111-1088-4280-9921-a7dcb5130971.webm)

---------------------

Test Case: [here](https://docs.google.com/spreadsheets/d/1jlA-uV_SzJFilKVvJm_8FMsiK320c12c162LWyPZA_8/edit#gid=483016940)
- Total Test Time: 1 hr
- Total Iterations: 1
- Total Comments: 1

**Pull request merged**